src/utils/DashboardHelperFunctions.js

```diff
@@ -47,6 +49,45 @@ export const getData = (oktaAxios, setUnapprovedIncidents) => {
     });
 };
 
+/**
+ * Returns a promise that resolves to an array of pending incidents
+ *
+ * @param {import('axios').AxiosInstance} oktaAxios
+ * @returns {Promise<any[]>} all pending incidents
+ */
+export function getPendingIncidents(oktaAxios) {
+  return oktaAxios.get('/dashboard/incidents')
+    .then(res => {
+      return res.data;
+    });
+}
+
+/**
+ * Returns a promise that resolves to an array of approved incidents
+ *
+ * @param {import('axios').AxiosInstance} oktaAxios
+ * @returns {Promise<any[]>} all approved incidents
+ */
+export function getApprovedIncidents(oktaAxios) {
+  return oktaAxios.get('/dashboard/incidents/approved')
+    .then(res => {
+      return res.data;
+    });
+}
+
+/**
+ * Returns a promise that resolves to an array of approved incidents
+ *
+ * @param {import('axios').AxiosInstance} oktaAxios
+ * @returns {Promise<any[]>} all approved incidents
+ */
+export function getFormResponses(oktaAxios) {
+  return oktaAxios.get('http://hrf-bw-labs37-dev.eba-hz3uh94j.us-east-1.elasticbeanstalk.com/to-approve')
```

Comment:
Does this url come out right? it kinda looks like it'll try to `get('https...backend_url...http://hrf-bw-labs37-dev.../to-approve')` since this is an oktaAxios instance with it's own base url.

Reply:
Good catch - honestly I just copied and pasted the link that was in the code already. Looking at the request in the dev tools, it does go through to the right URL. Axios must ignore the baseURL if you have the domain in the request URL.